cmake/configure/configure_2_trilinos.cmake

```diff
@@ -66,6 +66,15 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
       ENDIF()
     ENDFOREACH()
 
+    IF(${DEAL_II_TRILINOS_WITH_ROL})
+      IF(TRILINOS_VERSION VERSION_LESS 12.6.1)
+        MESSAGE(STATUS "ROL interface is disabled."
+          "deal.II will not support ROL interface if Trilinos version is less "
+          "than 12.6.1. Trilinos version found: \"${TRILINOS_VERSION}\".")
```

Comment:
Do you know if this is sharp?

Reply:
I am not sure about Trilinos@12.4.1 as I am not able to build it without errors so far. But Trilinos@12.6.1. But if someone could improve the sharpness.

Reply:
Let's not worry too much about sharpness. The people who have old versions of Trilinos already built did so likely without ROL. If someone wants to use ROL, they likely have to rebuild things, and then they may as well use the latest version.